agent/exec/containerd/adapter.go

```diff
@@ -71,6 +72,10 @@ type containerAdapter struct {
 	deleteResponse    *execution.DeleteResponse
 }
 
+func withNamespace(ctx context.Context) context.Context {
+	return namespaces.WithNamespace(ctx, "default")
```

Comment:
My plan was to make it a swarmd command line option (sooner rather than later, I just didn't get round to it). I'm not sure what the best default is.